### PR TITLE
Improve instrumentation

### PR DESCRIPTION
**Pull request by wagenet**
~Wait for #147.~

We should also document these events.

### CODE REVIEW

File: lib/graphiti/debugger.rb [hunk 1]
```diff
@@ -187,8 +187,8 @@ def flush_chunk(chunk, depth = 0)
     end
 
     ActiveSupport::Notifications.subscribe \
-      "graphiti.data", method(:on_data)
+      "data.graphiti", method(:on_data)
```
Comment (wadetandy) [inline lib/graphiti/debugger.rb]: Should change `data` out for `resolve` here.

Comment (wagenet) [reply lib/graphiti/debugger.rb]: Fixed.

File: lib/graphiti/scope.rb [hunk 2]
```diff
@@ -77,7 +77,7 @@ def broadcast_data
         sideload: @opts[:sideload],
         parent: @opts[:parent],
       }
-      Graphiti.broadcast("data", opts) do |payload|
+      Graphiti.broadcast(:"scope.resolve", opts) do |payload|
```
Comment (wadetandy) [inline lib/graphiti/scope.rb]: Sorry, one more thing.  `scope` is a bit of internal implementation detail in this case, so if we're expecting external consumption of this instrumentation I'd like to separate it a bit.  In `Debugger#on_data` we break out `sideload` vs `primary`, which I think is more clear. How would you feel about changing to that here?

Comment (wagenet) [reply lib/graphiti/scope.rb]: Fine with me :)

**Comment by wagenet** [comment]
@wadetandy looking around it seems like AS::N prefers `EVENT.LIBRARY` format with only two segments. At least that's what Rails does, even in cases where you could easily add another segment (e.g. `write_fragment.action_controller` and `read_fragment.action_controller`). If we stick with that, I guess we should do: `debug_flush.graphiti`, `render.graphiti`, `resolve_primary.graphiti`, `resolve_secondary.graphiti`.

**Comment by wadetandy** [comment]
Yeah that makes sense to me and I'm fine with that if it's what Rails, etc tends to do as well.

**Review by richmolj — APPROVED** [review]
LGTM, thanks @wagenet ❤️ !

**Pull request merged**